CHANGELOG.txt

```diff
@@ -1,5 +1,5 @@
 v6.31
-(XX/06/20)
+(05/06/20)
```

Comment:
I guess you would need to change date to 05.07.20

Reply:
Before I forget it: https://github.com/MichaIng/DietPi/commit/c7fd2a9b6904e2a531423311ad56ea1ddfb21034